index.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * Useful api:
+ * top headlines
+ * https://newsapi.org/v2/top-headlines?country=ca&apiKey=<key>
+ * - print total_results,
+ * - loop through articles and print article.title, article.description, article.urlToImage, article.publishedAt, article.url(to be opened in new tab)
+ */
+import news_api_key from './news_api_key.js';
+
+const api_key = news_api_key.api_key;
+
+const generateLi = (newsItem) => {
+  const li = document.createElement('li');
+  li.innerHTML = `<h2>${newsItem.title}</h2>
+   <h3> ${newsItem.description} </h3>
+   <img class="news__image" src=${newsItem.urlToImage} alt="Image missing" /><br/>
+   <a href=${newsItem.url} target="_blank"> Read More! </a>
```

Comment:
A recommendation I have is to remember that this is a blank tab they are on. Normally I would say this would be a best practice but this is a bit counter productive due to the page being a new tab that is  supposed to be filled.

Reply:
It makes sense @tdude . I didnt think that way. I think I recommended the opposite to you. :) But I agree here